tarball-ingester-iam.tf

```diff
@@ -207,6 +207,17 @@ data "aws_iam_policy_document" "tarball_ingester_describe_autoscaling" {
   }
 }
 
+data "aws_iam_policy_document" "tarball_ingester_set_desired_capacity_autoscaling" {
+  statement {
+    sid    = "AllowSetDesiredCapacityOnASG"
+    actions = [
+      "autoscaling:SetDesiredCapacity",
+    ]
+
+    resources = ["*"]
```

Comment:
Can this be limited to pointing at the ASG that is also generated in TF?

Reply:
done